src/content/blog/astro-i18n.mdx

```diff
@@ -0,0 +1,110 @@
+
+**This means every contributor, for every kind of contribution, uses the same platform and has the same contributing experience.** Instead of thinking of internationalization as some “other” thing, it’s a part of documentation. Our list of open PRs includes translations along with content updates and site infrastructure code fixes.
+
+To help people feel comfortable jumping in, and give them some clear direction on how to get started, we created a full Translation Guide in addition to our regular CONTRIBUTING.md. This contains not only the technical details about how our translations work but is also full of advice and tips for any beginner who might feel overwhelmed at the prospect of contributing.
+
+## Communication
+
+Our translators are not only fully integrated into our “docs as code” process, but also wider community discussions, and decision-making.
+
+Our translators coordinate and conspire on the Astro Discord server in their own `#docs-i18n` channel and dedicated language-specific threads. It's a special place where they are free to speak in (or about) their own language as they work on translations.
+
+At the same time, our translators are an active, engaged part of the wider Astro community and discussion, participating in general or off-topic conversations. You'll also often find them in more specific channels such as `#dev` or `#docs`, where thanks to their participation, we are able to deliver several internationalization improvements to both our docs and our core product from their feedback.
+
+And of course, you'll also find them dropping docs links and answering questions in our `#support` threads because they know the docs *so well* from having translated them!
+
+## Automation and Tooling
+
+It’s one thing to know *how* to get started. But, translators also need to know *where* to get started!
+
+Led by beloved Astro maintainer Hippo, our docs team built a custom [Translation Tracker](https://i18n.docs.astro.build/), the showpiece of our i18n automation! It tracks every docs page marked as "available to translate" by language and reports its own current state: 
+
+- fully-translated and up-to-date;
+- needing an update to match the English documentation;
+- or, not yet translated.
+
+<BlogContentImage src={trackerTable} alt="" />
+
+Internally, it uses the git history to compare changes between languages, defines the status of each page through some checks, and also generates a GitHub link with all the commits a page needs to be in sync with the English version. It’s powered by GitHub Actions, right out of our docs repo.
+
+<BlogContentImage src={trackerSummary} alt="" />
+
+Our Translation Tracker also uses GitHub data to list open translation PRs in need of reviews from native speakers, which are a great way to get started without the pressure of opening your own PR!
```

Comment:
I guess we're referring only to the Tracker here? So it should be "is" instead? Feel free to ignore if I am wrong!
```suggestion
Our Translation Tracker also uses GitHub data to list open translation PRs in need of reviews from native speakers, which is a great way to get started without the pressure of opening your own PR!
```

Reply:
We're referring to open PRs, They ARE a great way to get started!